File: app/src/main/java/org/mifos/mobile/ui/views/GuarantorListView.java
package org.mifos.mobile.ui.views;

/*
 * Created by saksham on 24/July/2018
 */

import org.mifos.mobile.models.guarantor.GuarantorPayload;
import org.mifos.mobile.ui.views.base.MVPView;

import java.util.List;

public interface GuarantorListView extends MVPView {

    void showGuarantorListSuccessfully(List<GuarantorPayload> list);
    void showError(String message);
}

File: app/src/main/java/org/mifos/mobile/ui/views/GuarantorListView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.models.guarantor.GuarantorPayload
import org.mifos.mobile.ui.views.base.MVPView

/*
* Created by saksham on 24/July/2018
*/   interface GuarantorListView : MVPView {
    fun showGuarantorListSuccessfully(list: List<GuarantorPayload?>?)
    fun showError(message: String?)
}